### ISSUE TITLE
Fix vanilla issue: Production panel shows incorrect completion time for wonders if there are multiple bonuses

### ISSUE THREAD

**Issue by bolbass** [CQUI-Org/cqui#563]
When a player has 2 bonuses to wonder production, Production panel uses only one of them to calculate turns to complete values. See more on [civfanatics](https://forums.civfanatics.com/threads/cqui-a-ui-overhaul-now-out-on-steam-workshop.602771/page-32#post-14795306).

**Comment by chaorace** [comment]
Is this an issue in vanilla as well?

**Comment by bolbass** [comment]
Yes, I checked it

**Comment by JHCD** [comment]
Is it an display error? Or does Vanilla not accept multiple bonuses?

**Comment by bolbass** [comment]
Yes, it's only a display error. In fact we build a wonder faster that it is shown in production panel when we have 2 bonuses.

**Comment by chaorace** [comment]
Reframing this as an enhancement since the issue comes from the game itself
